vsphere/vsphere.go

```diff
@@ -32,6 +32,7 @@ const (
 	vSphereDataStoreLock = "vsphere-ds-lock"
 	configProperty       = "config.hardware"
 	permissionError      = "Permission to perform this operation was denied"
+	svmotionErrorMsg     = "Retry pool expansion if you have performed svmotion on the disk"
```

Comment:
"retry pool expansion, if a storage vMotion operation was in progress during expansion"

Reply:
Done